com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Scene/ScreenNode.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor.ShaderGraph.GraphDelta;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class ScreenNode : IStandardNode
+    {
+        static string Name = "Screen";
+        static int Version = 1;
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+@"Width = ScreenParams.x;
+Height = ScreenParams.y;",
+            new ParameterDescriptor("Width", TYPE.Float, GraphType.Usage.Out),
+            new ParameterDescriptor("Height", TYPE.Float, GraphType.Usage.Out),
+            new ParameterDescriptor("ScreenParams", TYPE.Float, GraphType.Usage.Static, REF.ScreenParams)
```

Comment:
Check the data type on ScreenParams.  I think it at least needs to be a Vec2 because you're using it's x and y in the body code.